### ISSUE TITLE
Force New order on threads

### ISSUE THREAD

**Issue by zorbathut** [themotte/tildes#5]
As part of the "disable visible vote results" project, make sure that Best/Top sorting options don't exist; New-only.

**Comment by fjordelectro** [comment]
Do we want to completely remove the feature from tildes or provide a config option to force thread sorting globally? 

Similarly, what is our general philosophy when making breaking changes to tildes? Is this a hard fork or are we trying to upstream as much as possible?

**Comment by zorbathut** [comment]
@fjordelectro this got implemented before I answered your question, but the answer is . . . *well, it's complicated*.

We're trying to upstream as much as is reasonable, and we should spend a little extra effort on upstreamable options when that makes sense. If it requires a *lot* of extra effort, or is an option that is very specific to us (this particular feature is both of those) we currently shouldn't worry about it.

If we do end up using this as an escape hatch then we'll probably put the effort into revamping some of these features to be upstreamable, just to make downstreaming easier, but at the moment there's no guarantee this even ends up used.

So, tl;dr, use your best judgement, feel free to ask (here or on Discord) if you're unsure. :)
